tests/sqla_models_tests.py

```diff
@@ -223,6 +223,28 @@ def test_incorrect_jinja_syntax_raises_correct_exception(self):
             with pytest.raises(QueryObjectValidationError):
                 table.get_sqla_query(**query_obj)
 
+    def test_query_format_strip_trailing_semicolon(self): 
+        query_obj = {
+            "granularity": None,
+            "from_dttm": None,
+            "to_dttm": None,
+            "groupby": ["user"],
+            "metrics": [],
+            "is_timeseries": False,
+            "filter": [],
+            "extras": {},
+        }
+
+        # Table with Jinja callable.
+        table = SqlaTable(
+            table_name="test_table",
+            sql="SELECT * from test_table;",
+            database=get_example_database(),
+        )
+        sqlaq = table.get_sqla_query(**query_obj)
+        sql = get_example_database().compile_sqla_query(sqlaq.sqla_query)
+        assert sql.match("SELECT * from test_table")
```

Comment:
I assume this should be a regular equals assert:
```suggestion
        assert sql == "SELECT * from test_table"
```